chromium_src/third_party/blink/renderer/modules/mediastream/media_devices.cc

```diff
@@ -26,15 +26,19 @@ namespace brave {
 void FarbleMediaDevices(ExecutionContext* context,
                         MediaDeviceInfoVector* media_devices) {
   WebContentSettingsClient* settings = GetContentSettingsClientFor(context);
```

Comment:
settings can be placed after the media_devices size check.

Reply:
Missed this before I merged. Followup here: https://github.com/brave/brave-core/pull/10425